src/frequenz/client/reporting/_client.py

```diff
@@ -172,7 +172,7 @@ async def list_single_component_data(
             metrics: The metric name or list of metric names.
             start_dt: The start date and time.
             end_dt: The end date and time.
-            resolution: The resampling resolution for the data, represented in seconds.
+            resampling_period: The period for resampling the data, specified as `timedelta` object.
```

Comment:
No need to change it now, but for future, I think the extra type info isn't required in the comment, the type hints already give us that :)

Reply:
Good point! I have taken it out since I had to rebase on v0.x.x to get rid of the merge commit in any case.